native/src/http_client.c

```diff
@@ -174,10 +183,11 @@ static void s_stream_on_incoming_body(
     size_t *out_window_update_size,
     void *user_data) {
     (void)s;
-    (void)out_window_update_size;
     struct aws_dotnet_http_stream *stream = user_data;
     if (stream->on_incoming_body) {
-        stream->on_incoming_body(data->ptr, data->len);
+        uint64_t window_update_size = *out_window_update_size;
+        stream->on_incoming_body(data->ptr, data->len, &window_update_size);
+        *out_window_update_size = window_update_size;
```

Comment:
is this necessary for super-safety?
`*out_window_update_size = window_update_size < SIZE_MAX ? window_update_size : SIZE_MAX;`

Reply:
Always be safe, kids.